dallinger/experiment_server/dashboard.py

```diff
@@ -564,6 +565,17 @@ def mturk():
     return render_template("dashboard_mturk.html", title="MTurk Dashboard", data=data)
 
 
+@dashboard.route("/auto_recruit/<bool_val>", methods=["POST", "GET"])
```

Comment:
Can you set this to just POST? If you want to implement a GET route that's fine but a GET route should just return the current value of auto_recruit.

Reply:
Why? I think it's useful to support both POST and GET. GET will be useful as it allows the experimenter to conveniently set the auto_recruit value even if the monitor crashes. Or do you have other concerns?

Reply:
This is just how the definition of GET/POST requests works, see e.g. https://www.w3schools.com/tags/ref_httpmethods.asp

> GET is used to request data from a specified resource.
> POST is used to send data to a server to create/update a resource.

I think it's bad practice to deviate unnecessarily from these.

Reply:
I did some more reading and a concrete issue is that GET requests can be cached, which is problematic here